do

```diff
@@ -50,12 +52,52 @@ def update_foretold(args):
         log.info(f"Saved Foretold to {dest}")
 
 
+def _web_export(
+    comment: str,
+    export_regions: List[str],
+    region_dataset: RegionDataset,
+    models: Path,
+    rates: Path,
+    hopkins: Path,
+    foretold: Path,
+    output_dir: Path,
+    date_resample: str,
+) -> None:
+    ex = WebExport(date_resample, comment=comment)
+
+    simulation_specs: pd.DataFrame = pd.read_hdf(models, "simulations")
+    models_df: pd.DataFrame = pd.read_hdf(models, "new_fraction")
+    # TODO: unify indexing
+    rates_df: pd.DataFrame = pd.read_csv(rates, index_col="CodeM49")
+    hopkins_df: pd.DataFrame = pd.read_csv(hopkins, index_col="Code", parse_dates=["Date"])
+    foretold_df: pd.DataFrame = pd.read_csv(foretold, index_col="Code", parse_dates=["Date"])
+
+    for code in export_regions:
+        reg = region_dataset[code]
+        ex.new_region(
+            reg,
+            models_df.loc[code].sort_index(level="Date"),
+            simulation_specs,
+            rates_df.loc[int(reg.M49Code)],
+            hopkins_df.loc[code].set_index('Date').sort_index(),
+            foretold_df.loc[code].set_index('Date').sort_index(),
+        )
+
+    ex.write(output_dir)
+
+
 def web_export(args):
-    ex = WebExport(comment=args.comment)
-    for code in args.config["export_regions"]:
-        ex.new_region(args.rds[code])
-    # TODO: add data to ex
-    ex.write(args.config["output_dir"])
+    _web_export(
```

Comment:
Why the wrapper?

Reply:
To know explicitly what's actually being used and not relying on global state of `args`.

Reply:
I respect your aesthetic, but you create a 11-line function that only calls another function with 9 positional arguments (most paths, so easily swapped), and then just unpack these just 35 lines above ... :grin:

Reply:
Don't worry, I will remove it for you :-P .

Reply:
Also, `_web_export` can be much nicely tested and has a typehinted signature. By removing the wrapper, readability suffers.

Reply:
Addressed in: https://github.com/epidemics/epimodel/pull/25/commits/8fef0dd1b9fc52b38f0ca37aafc61c7f7f6e8842

I admit that I hate it this way, it's imo much less reader-friendly as they have to parse the whole function to know what's being used